### PR TITLE
normalized the return value of Album/Settings getThumbnailSize

### PR DESCRIPTION
**Pull request by jochenmanz**
**Why:**

When no settings are present, the current behaviour will return a array with an empty value instead of an empty array. This makes it difficult to check if there are sizes as the array needs to be analysed.

if (empty($sizes) vs if (empty($sizes[0]))

**What:**

Changed the return value to return a empty array if its empty.

**Breaking something?**

Anything that relies on the old format might break

### CODE REVIEW

File: tests/Functional/Models/Album/SettingsTest.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?php
```
Comment (bcremer) [inline tests/Functional/Models/Album/SettingsTest.php]: This test should be moved into the `Unit` directory as it has no external dependencies.
You can run the Unit tests separatly using the `composer test-unit` command (`./vendor/bin/phpunit -c tests/phpunit_unit.xml.dist`)

File: tests/Functional/Models/Album/SettingsTest.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ * "Shopware" is a registered trademark of shopware AG.
+ * The licensing of the program under the AGPLv3 does not imply a
+ * trademark license. Therefore any rights, title and interest in
+ * our trademarks remain entirely with us.
+ */
+
+/**
+ * @category  Shopware
+ * @package   Shopware\Tests
+ * @copyright Copyright (c) 2014, shopware AG (http://www.shopware.de)
```
Comment (bcremer) [inline tests/Functional/Models/Album/SettingsTest.php]: Old header.

``` diff
- * @copyright Copyright (c) 2014, shopware AG (http://www.shopware.de)
+ * @copyright Copyright (c) shopware AG (http://www.shopware.de)
```

File: tests/Functional/Models/Album/SettingsTest.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+{
+    public function testGetThumbnailSizeReturnValue()
+    {
+        $settings = new \Shopware\Models\Media\Settings();
+        $settings->setThumbnailSize([]);
+
+        $size = $settings->getThumbnailSize();
+
+        $this->assertEquals([], $size);
+        $this->assertNotEquals([0 => ''], $size);
```
Comment (bcremer) [inline tests/Functional/Models/Album/SettingsTest.php]: Please remove the `$this->assertNotEquals` line. The result cannot equal `[]` and not-not equal `[0 => '']`. 
I suppose this line was there to show the old behavior.

**Comment by jochenmanz** [comment]
Hi,

thanks for the feedback, i just added the requested changes.

Best regards,
Jochen

File: tests/Unit/Model/Album/SettingsTest.php [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+ * trademark license. Therefore any rights, title and interest in
+ * our trademarks remain entirely with us.
+ */
+
+/**
+ * @category  Shopware
+ * @package   Shopware\Tests
+ * @copyright Copyright (c) shopware AG (http://www.shopware.de)
+ */
+class Shopware_Tests_Models_Album_SettingsTest extends Enlight_Components_Test_TestCase
```
Comment (bcremer) [inline tests/Unit/Model/Album/SettingsTest.php]: This does not work when running the unit tests separatly with `composer test-unit`

``` diff
- Enlight_Components_Test_TestCase
+ PHPUnit_Framework_TestCase
```

**Pull request closed without merging**